### PR TITLE
fix(volumeslider): check for TouchEvent to disallow mute on click on …

### PR DESCRIPTION
**Pull request by ivo-toby**
…mobile. Small fix for mobile on which the volume slider is not hiding

### CODE REVIEW

File: src/js/standard/VolumeSlider/index.js [hunk 1]
```diff
@@ -64,7 +64,10 @@ class VolumeSlider extends BaseElement {
         this.on('playerLoadedMetadata', () => this.onVolumeChange());
 
         // Add mouse event listeners
-        this.volumeButton.addEventListener('click', () => this.onClick());
+        if (!window.TouchEvent) {
```
Comment (Thomasvdam) [inline src/js/standard/VolumeSlider/index.js]: How about those laptops that also have touch capabilities?

**Review by Thomasvdam — CHANGES_REQUESTED** [review]
I think we'll run into problems with laptops that have touchscreens. They usually support both event types.

File: src/js/standard/VolumeSlider/index.js (hunk 1 above)
Comment (ivo-toby) [reply src/js/standard/VolumeSlider/index.js]: Good point

**Comment by ivo-toby** [comment]
Can someone please check this again ?

**Review by Thomasvdam — APPROVED** [review]
Lgtm. I'd squash the two commits when rebasing. :)

**Pull request merged**